src/helpers/messages.js

```diff
@@ -40,9 +40,24 @@ function getExitMessage(scope, runner, message = '', sync) {
         itemsCountMessage = getUnansweredItemsWarning(scope, runner, sync);
     }
 
-    return `${itemsCountMessage} ${message}`.trim();
+    return `${getHeader(scope)}${itemsCountMessage} ${message}`.trim();
+}
+/**
+ * Build message if not all items have answers
+ * @param {String} scope - scope to consider for calculating the stats
+ * @param {Object} runner - testRunner instance
+ * @param {Boolean} sync - flag for sync the unanswered stats in exit message and the unanswered stats in the toolbox. Default false
+ * @returns {String} Returns the message text
+ */
+function getHeader(scope) {
+    if (scope === 'section' || scope === 'testSection') {
+        return `<b>${__('You are about to leave this section.')}</b><br><br>`;
+    } else if (scope === 'test' || scope === 'testWithoutInaccessibleItems') {
+        return `<b>${__('You are about to submit the test.')}</b><br><br>`;
+    } else if (scope === 'part') {
+        return `<b>${__('You are about to submit this test part.')}</b><br><br>`;
+    }
```

Comment:
I realize, this shouldn't event be a case, but let's have a default `return` value, just to be fully compliant with the method's contract.
```suggestion
    }

    return '';
```